curriculum/challenges/english/25-front-end-development/quiz-html-tables-and-forms/66ed902df45ce3ece4053ebc.md

```diff
@@ -17,439 +17,479 @@ To pass the quiz, you must correctly answer at least 17 of the 20 of the questio
+`tc`
 
 ---
 
-Placeholder distractor 3
+`tcol`
 
 #### --answer--
 
-Placeholder answer
+`td`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What defines the cell of a header and improves accessibility?
 
 #### --distractors--
 
-Placeholder distractor 1
+`colspan`
 
 ---
 
-Placeholder distractor 2
+`style`
 
 ---
 
-Placeholder distractor 3
+`table` with attribute `style = " "`
 
 #### --answer--
 
-Placeholder answer
+`scope`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `<input>` tag do?
 
 #### --distractors--
 
-Placeholder distractor 1
+It allows users to enter any type of data.
 
 ---
 
-Placeholder distractor 2
+It is a pop-up box that asks user to enter data.
 
 ---
 
-Placeholder distractor 3
+It can be only used inside a form to enter user input.
 
 #### --answer--
 
-Placeholder answer
+It defines a single line text field by default.
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to use `<input>` inside a form to properly send data?
 
 #### --distractors--
 
-Placeholder distractor 1
+```html
+<form>
+    <input type="text" name="username" placeholder="Enter your username"></input>
+    <input type="submit" value="Submit"></input>
+</form>
+```
 
 ---
 
-Placeholder distractor 2
+```html
+<form>
+    <input type="text" name="username" placeholder="Enter your username" type="submit" value="Submit">
+</form>
+```
 
 ---
 
-Placeholder distractor 3
+All of the other choices are correct.
 
 #### --answer--
 
-Placeholder answer
+```html
+<form>
+    <input type="text" name="username" placeholder="Enter your username">
+    <input type="submit" value="Submit">
+</form>
+```
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which attribute specifies that the element must be filled out before submitting the form?
 
 #### --distractors--
 
-Placeholder distractor 1
+`necessary`
 
 ---
 
-Placeholder distractor 2
+`imp`
 
 ---
 
-Placeholder distractor 3
+It is not possible via HTML only.
 
 #### --answer--
 
-Placeholder answer
+`required`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How to hide an input from the user?
 
 #### --distractors--
 
-Placeholder distractor 1
+`<hidden></hidden>`
 
 ---
 
-Placeholder distractor 2
+`<input type="text" hidden>`
 
 ---
 
-Placeholder distractor 3
+It is not possible via HTML.
 
 #### --answer--
 
-Placeholder answer
+`<input type="hidden" name="inputName" value="inputValue">`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `target` attribute do?
 
 #### --distractors--
 
-Placeholder distractor 1
+It specifies the variable we intend to change.
 
 ---
 
-Placeholder distractor 2
+It specifies different ways to store the response of the form.
 
 ---
 
-Placeholder distractor 3
+It specifies different keyword searches.
 
 #### --answer--
 
-Placeholder answer
+It specifies a name or keyword that indicates where to display the response after submitting the form.
```

Comment:
```suggestion
It specifies indicates where to display the response after submitting the form.
```

Reply:
Can I alter it as below?
It indicates where to display the response after submitting the form.